### PR TITLE
add ability to add documentation links to feature flags

### PR DESCRIPTION
**Pull request by czue**
image

@snopoke @kaapstorm

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 
 
 class StaticToggle(object):
-    def __init__(self, slug, label, namespaces=None):
+    def __init__(self, slug, label, namespaces=None, documentation_link=None):
```
Comment (benrudolph) [inline corehq/toggles.py]: minor, but feature preview uses `help_link`. may be better to keep it consistent to reduce any dumb bugs when migrating from toggle to feature preview

**Comment by czue** [comment]
oh good call, i'll reconcile those two concepts.

**Comment by czue** [comment]
@benrudolph updated

**Comment by benrudolph** [comment]
:+1:

**Pull request merged**